lab_6_pipeline/pipeline.py

```diff
@@ -19,10 +43,22 @@ def __init__(self, path_to_raw_txt_data: Path):
         Initializes CorpusManager
         """
 
+        self._path = path_to_raw_txt_data
+        self._storage = []
+        self._validate_dataset()
+
     def _validate_dataset(self) -> None:
         """
         Validates folder with assets
         """
+        if not self._path.exists():
+            raise FileNotFoundError
+        if not self._path.is_dir():
+            raise NotADirectoryError
+        if len(meta_files) != len(raw_files):
```

Comment:
Where do you define these variables?

Reply:
i'm so sorry just saw this kind of check but did it badly
now I've done it in a better way